core-services/sync-rover-groups/_config.yaml

```diff
@@ -14,6 +14,10 @@ groups:
   aos-qe:
     clusters:
     - app.ci
+    - build01
```

Comment:
@smg247 For this [openshift/openshift-tests-private](https://github.com/openshift/openshift-tests-private) Prow test, seems like it uses the `build03` cluster, such as https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_openshift-tests-private/2689/pull-ci-openshift-openshift-tests-private-master-e2e-aws/1493230897246244864
So, I add all `build0*` cluster here. Could you help have a review when you get a chance? Thanks!

Reply:
Once this PR merge, the cluster will point to `qe-priovate-deck`, such as  https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/, so no need the `build01-04` clusters here. cc: @kuiwang02